readme/api/references/development_mode.md

```diff
@@ -1,5 +1,5 @@
 # Development mode
 
-When experimenting with Joplin, for example when developing a plugin or trying a theme, you might want to run Joplin in development mode. Doing so means that Joplin will run using a different profile, so you can experiment without risking to accidentally change or delete your data.
+When experimenting with Joplin, for example when [developing a plugin](https://joplinapp.org/api/get_started/plugins/) or trying a theme, you might want to run Joplin in development mode. Doing this will open a second copy of Joplin using a different profile with test notes and notebooks, so you can experiment without risking changing or deleting your data.
 
-To enable Development Mode, open Joplin as normal, then go to **Help => Copy dev mode command to clipboard**. This will copy a command to the clipboard. Now close Joplin, and start it again in dev mode using the command you've just copied.
\ No newline at end of file
+To enable Development Mode, go to `Help => Copy dev mode command to clipboard`. This will copy a command to your clipboard. Paste this command in a shell / terminal to run a development version of the app. 
```

Comment:
Any reason why the command was moved from bold formatting to code formatting? You might be right about this actually but I don't know. How is it in other documentations for things like menu item paths? If code block is what's usually used then why not.